### ISSUE TITLE
ISSUE: As a fan I should be able to Comment on a single post? *

### ISSUE THREAD

**Issue by ILikeConcerts** [CrouchJumpInc/fangagement-public#22]
Wrote a comment on a post on the timeline, then opened the post I commented on so I could view it as a single post and my comment from the timeline does not load into the comment thread, but when I back out to the timeline my comment is there. I then go back into the single post (comment still not viewable) and decide to write a new comment, and it pops up in the comment thread during the time I'm viewing the single post. I then back out to timeline view and the comment just I wrote in the single post view is not viewable and was not loaded back in when I revisited the single post view. 

TL;DR comment from timeline only viewable in timeline, single post comments arent viewable in the timeline, and don't reload back when visiting single post mode.

tested on 2 separate posts on the timeline

**Comment by ILikeConcerts** [comment]
*update* the comments created from a single post view load into the timeline, but I had to log into another account to be able to view the comments. Seems it might be an issue for the account posting, but others can still view it

**Comment by DrVonDevious** [comment]
I believe the comments should show up after reloading the app since it is probably due to the app not refreshing posts. created FANS-137 to look into this
